docs/index.md

```diff
@@ -90,7 +90,12 @@ The `remote` block supports:
 
 * `name` - *Optional* - The name of the LXD remote.
 
-* `password` - *Optional* - The password to authenticate to the LXD remote.
+* `password` - *Optional* - The [trust password](https://documentation.ubuntu.com/lxd/en/latest/authentication/#adding-client-certificates-using-a-trust-password)
+  to authenticate to the LXD remote. This method is **not recommended** and will most likely be
+  removed from LXD in the future releases. Use `token` instead.
+
+* `token` - *Optional* - The [token](https://documentation.ubuntu.com/lxd/en/latest/authentication/#adding-client-certificates-using-tokens)
+  to authenticate to the LXD remote.
```

Comment:
I wouldn't say it is used to authenticate to the LXD remote. It's more to securely add the LXD remote locally (due to the embedded fingerprint) and also to inject the client certificate into LXD's trust store.